bindings/python/docs/source/extension_types.rst

```diff
@@ -0,0 +1,40 @@
+Extension Types
+===============
+
+This tutorial is intended as an introduction to working with
+**PyMongoArrow** and its corresponding extension types. The reader is assumed to be familiar with basic
+`PyMongo <https://pymongo.readthedocs.io/en/stable/tutorial.html>`_ and
+`MongoDB <https://docs.mongodb.com>`_ concepts.
+
+Extension types with Arrow
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Both extension types, :class:`pymongoarrow.types.ObjectIdType` and :class:`pymongoarrow.types.Decimal128StringType`, are only partially supported in PyArrow. They will work when used in a
+schema, but will show up in the table as a `fixed_size_binary(12)` or `string` respectively::
+
+        schema = Schema({"_id": ObjectIdType(), "data": Decimal128StringType()})
+        table = find_arrow_all(coll, {}, schema=schema)
+        print(table)
+        >>> pyarrow.Table
+        >>> _id: fixed_size_binary[12]
+        >>> data: string
+        >>> ----
+        >>> _id: [[63C003BF0A1D5281D33B0AFD,63C003BF0A1D5281D33B0AFE,63C003BF0A1D5281D33B0AFF,63C003BF0A1D5281D33B0B00]]
+        >>> data: [["0.1","1.0","0.00001",null]]
+        >>> ...
+
+
+
+Extension types with Pandas/NumPy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Extension types with Pandas/NumPy are only partially supported. They will work when used in a
+schema, but will show up in the table as a :class:`pandas.object`::
```

Comment:
I think we should also say that the underlying value is ``fixed_size_binary(12)` or `string` as above.

Reply:
In Pandas/NumPy that is not the case. The underlying type is object, because we don't have pandas extension types like with Arrow.

Reply:
The array type is object, but when you access a field it will be a python object.

Reply:
Perhaps I'm misunderstanding, but isn't array type == "will show up in the table as"?

Reply:
That makes sense. Added.